### PR TITLE
Share common Mime related classes

### PR DESCRIPTION
**Pull request by aduprat**
Not a real review, only sharing work.

### CODE REVIEW

File: mailbox/common/src/main/java/org/apache/james/mailbox/message/json/MixInMessageUpdateJson.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.message.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public abstract class MixInMessageUpdateJson {
+
+
```
Comment (chibenwa) [inline mailbox/common/src/main/java/org/apache/james/mailbox/message/json/MixInMessageUpdateJson.java]: Extra line

**Comment by chibenwa** [comment]
Why not in james-project/mailbox/store/src/main/java/org/apache/james/mailbox/store/json ???

**Comment by chibenwa** [comment]
Other question : I am interrested to know which other part of James will use this functionnality :-)

**Comment by aduprat** [comment]
1/ We need Java-8 for some moved classes and mailbox-store is only Java-6
2/ It's shared between ES & Jmap, and we will need it again in order to store the mails attachments in Cassandra (new API, table)

File: mailbox/common/src/main/java/org/apache/james/mailbox/message/json/MixInMessageUpdateJson.java (hunk 1 above)
Comment (aduprat) [inline mailbox/common/src/main/java/org/apache/james/mailbox/message/json/MixInMessageUpdateJson.java]: Done

**Pull request closed without merging**